CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.11.0]
```

Comment:
We already have a 0.11.0 release (we currently are at 0.12.1 actually). I think this changelog file was not updated. You can use `0.12.2` instead.